android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -179,6 +189,30 @@ public ReactExoplayerView(ThemedReactContext context, ReactExoplayerConfig confi
         themedReactContext.addLifecycleEventListener(this);
         audioBecomingNoisyReceiver = new AudioBecomingNoisyReceiver(themedReactContext);
 
+        ReactExoplayerView self = this;
+
+        BroadcastReceiver pipReceiver = new BroadcastReceiver() {
```

Comment:
can this be an optional behavior? maybe props based?

Reply:
Of course, good idea!

Reply:
Well, actually it's an essential part of the implementation which is required to make it work. In this case, pipReceiver is responsible for PIP mode changes, so if user returns to the regular app (press maximise button), this receiver notifies videoEventEmitter which afterwards trigger onPictureInPictureStatusChanged callback to notify JS about such a change. Without this, client would never be able to detect whether the app is running in PIP mode or not.

But I can of course create the other receiver prop driven. That one is responsible for triggering PIP mode when user press Home or Recent button, which should be optional indeed.